app/bundles/EmailBundle/Tests/Helper/MailHelperTest.php

```diff
@@ -1,5 +1,4 @@
 <?php
-
```

Comment:
It's failing because the files aren't following the Symfony Code Standards. Travis is showing you where new lines should be with `+` sign: https://travis-ci.org/mautic/mautic/jobs/371794255#L706

Good rule is to touch only the code necessary for your bug fix or feature. Leave the rest be. It will also speed up the code review. So you'll  have to fix the code style check.